test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -444,15 +446,23 @@
         (is (thrown-with-msg?
               IllegalArgumentException
               #"Invalid script string in webserver 'post-config-script' configuration"
-              (jetty9/start-webserver!
-                (jetty9/initialize-context)
-                (merge base-config
-                       {:post-config-script (str "AHAHHHGHAHAHAHEASD!  OMG!")})))))
+              (let [context (jetty9/initialize-context)]
+                (try
+                  (jetty9/start-webserver!
+                   context
+                   (merge base-config
+                          {:post-config-script (str "AHAHHHGHAHAHAHEASD!  OMG!")}))
+                  (finally
+                    (jetty9/shutdown context)))))))
       (testing "Throws an error if the script can't be executed."
         (is (thrown-with-msg?
               IllegalArgumentException
               #"Invalid script string in webserver 'post-config-script' configuration"
-              (jetty9/start-webserver!
-                (jetty9/initialize-context)
-                (merge base-config
-                       {:post-config-script (str "Object x = null; x.toString();")}))))))))
+              (let [context (jetty9/initialize-context)]
+                (try
+                  (jetty9/start-webserver!
+                   context
+                   (merge base-config
+                          {:post-config-script (str "Object x = null; x.toString();")}))
+                  (finally
+                    (jetty9/shutdown context))))))))))
```

Comment:
I think there's also one from https://github.com/cprice404/trapperkeeper-webserver-jetty9/blob/bug/master/TK-301-fix-HUP-memleaks/test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj#L760 but I can't comment on that in the diff of this PR easily.